tests/list_of_tests.cmake

```
@@ -1962,12 +1962,12 @@ four_c_test_nested_parallelism(sohex8_multiscale_macro_2micro.dat sohex8_multisc
 four_c_test_nested_parallelism(tsi_heatconvection_monolithic.dat tsi_heatconvection_monolithic.dat "")
 
 # testing the whole framework: from pre_exodus and 4C to the post-filter:
-four_c_test_framework(tutorial_battery 2 "")
-four_c_test_framework(tutorial_contact_3d 2 "")
-four_c_test_framework(tutorial_fluid 2 "")
-four_c_test_framework(tutorial_fsi 2 "")
-four_c_test_framework(tutorial_fsi_3d 2 fsi_part_struct_solver.xml)
-four_c_test_framework(tutorial_solid 3 xml/multigrid/elasticity_template.xml)
+four_c_test_framework(PREFIX tutorial_battery NP 2)
+four_c_test_framework(PREFIX tutorial_contact_3d NP 2)
+four_c_test_framework(PREFIX tutorial_fluid NP 2)
+four_c_test_framework(PREFIX tutorial_fsi NP 2)
+four_c_test_framework(PREFIX tutorial_fsi_3d NP 2 COPY_FILES ${PROJECT_SOURCE_DIR}/tests/input_files/fsi_part_struct_solver.xml)
```

why do we want to fully specify the path to the copy file here?

In my opinion we should stay consistent with all file paths

the regular test files do also assume the path to the input file folder?


```suggestion
four_c_test_framework(PREFIX tutorial_fsi_3d NP 2 COPY_FILES fsi_part_struct_solver.xml)
```

---

I thought about the case where you put the support file in the `framework_tests` directory.

---

That being said, yes, let's try to be more consistent with the paths. Maybe this is something for your planned cleaning of these helper functions?

---

To be fully path independent of the source directory, we would need to copy all test files to the build directory.

---

> I thought about the case where you put the support file in the framework_tests directory.

Is this planned?

But I am fine with the current solution - if one has ideas on a better approach regarding our test paths please add it to #56